src/Authentication/Controllers/RequestSystemUserController.cs

```diff
@@ -227,4 +228,34 @@ public async Task<ActionResult<CreateRequestSystemUserResponse>> ApproveSystemUs
 
         return NotFound();
     }
+
+    /// <summary>
+    /// Retrieves a list of Status-Response-model for all Requests that the Vendor has for a given system they own.
+    /// </summary>
+    /// <param name="systemId">The system the Vendor wants the list for</param>
+    /// <param name="cancellationToken">The cancellation token</param>
+    /// <returns>Status response model CreateRequestSystemUserResponse</returns>
+    [Authorize(Policy = AuthzConstants.POLICY_SCOPE_SYSTEMREGISTER_WRITE)]
+    [HttpGet("vendor/bysystem/{systemId}")]
+    public async Task<ActionResult<CreateRequestSystemUserResponse>> GetAllRequestsForVendor(string systemId, CancellationToken cancellationToken = default)
```

Comment:
The model name "CreateRequestSystemResponse" can be changed to something like "RequestSystemResponse" to be more generic

Reply:
agreed, changed in update commit